.github/workflows/release-backwards-compatiblity.yml

```diff
@@ -21,10 +22,33 @@ env:
   MESSAGING_USER_ARN: ${{secrets.MESSAGING_USER_ARN}}
 
 jobs:
+  check-backward-compatibility-run:
+    name: Check if backward compatibility check is needed for the version we want to release
+    runs-on: ubuntu-latest
+    outputs:
+      is-beta: ${{ steps.check-if-beta-version.outputs.is-beta }}
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check if beta version
+        id: check-if-beta-version
+        run: |
+          current_version=$(.github/script/get-current-version)
```

Comment:
I dont think this is entirely correct. We dont want to run this for the first official version either (e.g., v3.0.0 as no prev major exist).

Reply:
Yes I have not added any checks just checking with what was existing, was refreshing some GitHub actions stuff doing it. But yes, 3.0.0 or any first major version needs to be skipped.

Reply:
Fixed now.